base/mysqlxx/Pool.cpp

```diff
@@ -125,25 +125,37 @@ Pool::~Pool()
 
 Pool::Entry Pool::get()
 {
+    auto & logger = Poco::Util::Application::instance().logger();
+
     std::unique_lock<std::mutex> lock(mutex);
 
     initialize();
     for (;;)
     {
+        logger.debug("Iterating through existing MySQL connections for %s", getDescription());
```

Comment:
Probably, we should remove logging.
Or use `trace` and add some description that message is from mysql connection pool (also to all messages below)

Reply:
Logging has proved itself to be very important when debugging mysql connection related problems. Although it makes sense to raise the level of verbosity to `trace`.